client/js/components/Navigation.jsx

```diff
@@ -3,12 +3,14 @@ import React from 'react';
 
 export default class Navigation extends React.Component {
     render () {
+        var h1 = <h1 onClick={() => window.location.reload()}>Austin Green Map</h1>;
```

Comment:
I'll rename it to title.

I'm not sure why you would use `const` here, maybe @prestonp can answer?